Python/Product/Uwp/Project/PythonUwpPropertyPage.cs

```diff
@@ -32,6 +32,16 @@ class PythonUwpPropertyPage : CommonPropertyPage {
         public PythonUwpPropertyPage() {
             _control = new PythonUwpPropertyPageControl(this);
         }
+
+        public void Dispose() {
```

Comment:
I'm wondering who calls this one as well, but the protected pattern is also certainly not necessary